includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -21,7 +21,21 @@ abstract class WC_Stripe_Payment_Gateway extends WC_Payment_Gateway_CC {
 	 */
 	public function display_admin_settings_webhook_description() {
 		/* translators: 1) webhook url */
-		return sprintf( __( 'You must add the following webhook endpoint <strong style="background-color:#ddd;">&nbsp;%s&nbsp;</strong> to your <a href="https://dashboard.stripe.com/account/webhooks" target="_blank">Stripe account settings</a>. This will enable you to receive notifications on the charge statuses.', 'woocommerce-gateway-stripe' ), WC_Stripe_Helper::get_webhook_url() );
+		$must_add = sprintf( __( 'You must add the following webhook endpoint <strong style="background-color:#ddd;">&nbsp;%s&nbsp;</strong> to your <a href="https://dashboard.stripe.com/account/webhooks" target="_blank">Stripe account settings</a>. This will enable you to receive notifications on the charge statuses.', 'woocommerce-gateway-stripe' ), WC_Stripe_Helper::get_webhook_url() );
```

Comment:
How about adding "If you haven't already done so..." - low volume merchants might not get a webhook for some time after setting this

Reply:
This string isn't new in this PR (pointing out what you likely already know), but that sounds like a good idea – done in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1224/commits/b6d8f62866ca4cf0844e3b5c85e9e1bc39a92bdf, with wording tweaked to imply it needs to currently be enabled – what do you think?

We can think about showing a different string if an event has already been received, but it's probably not necessary.